ambry-server/src/main/java/com/github/ambry/server/AmbryServerRequests.java

```diff
@@ -778,7 +778,6 @@ private ServerErrorCode handleRemoveStoreRequest(PartitionId partitionId) throws
     Store store = ((StorageManager) storeManager).getStore(partitionId, true);
     // Attempt to remove store from storage manager.
     if (storeManager.removeBlobStore(partitionId) && store != null) {
```

Comment:
at some point we have to re-evaulate if these admin command is still valid or not. This goes against dynamic helix replica management. I always thought this is still there because we still support static cluster map. But I might be wrong. If this is not used anywhere, then we should retire these logics.

Reply:
Sure. I will check with SRE if we still use admin commands for store addition and removal. If not, I will put up a PR to remove them.